ci/script_multi.sh

```diff
@@ -32,8 +33,8 @@ elif [ "$COVERAGE" ]; then
 
 elif [ "$SLOW" ]; then
     TEST_ARGS="--only-slow --skip-network"
-    echo pytest -m "not single and slow" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
-    pytest      -m "not single and slow" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
+    echo pytest -n 2 -m "not single" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
+    pytest      -n 2 -m "not single" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
```

Comment:
This is in `script_multi.py`, so I assume for the slow tests with also want to test with two processes, like in the others.

I don't think the `slow` in the `-m` is needed, as we already have the `--only-slow` in `TEST_ARGS` that is the way we call the slow tests everywhere else (and it's handled in `conftest.py`

Reply:
you can try but this has issues before iirc